### PR TITLE
Implement fee monitor

### PR DESCRIPTION
**Pull request by foriequal0**
Resolves: https://github.com/CodeChain-io/cron-jobs/issues/9

### CODE REVIEW

File: fee-monitor/.eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-unused-vars": ["error", {
+      "argsIgnorePattern": "^_",
+      "varsIgnorePattern": "^_",
+      "caughtErrorsPattern": "^_"
+    }]
+  }
+}
```
Comment (sgkim126) [inline fee-monitor/.eslintrc.json]: You set insert_final_newline.

File: fee-monitor/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    }
+    tracer.deposit(author, tracer.totalFee.minus(distributed));
+}
+
+async function main() {
+    let lastBlockNumber = 0;
+
+    for (;;) {
+        const bestBlockNumber = await sdk.rpc.chain.getBestBlockNumber()!;
+        if (bestBlockNumber - lastBlockNumber > 1) {
```
Comment (sgkim126) [inline fee-monitor/src/index.ts]: Do we need these logs?

File: fee-monitor/src/Stake.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+function decodePlatformAddress(buffer: Buffer): PlatformAddress {
+    const accountId = buffer.toString("hex");
+    return PlatformAddress.fromAccountId(accountId, {
+        networkId: sdk.networkId,
+    });
+}
+
+export async function getStakeholders(blockNumber: number): Promise<PlatformAddress[]> {
+    const data = await sdk.rpc.engine.getCustomActionData(2, ["StakeholderAddresses"], blockNumber);
+    if (data == null) {
```
Comment (sgkim126) [inline fee-monitor/src/Stake.ts]: When it becomes null?
Isn't it an error?

File: fee-monitor/src/Stake.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    return RLP.decode(Buffer.from(data, "hex")).map(decodePlatformAddress);
+}
+
+async function getCCSBalance(address: PlatformAddress, blockNumber: number): Promise<U64> {
+    const data = await sdk.rpc.engine.getCustomActionData(
+        2,
+        ["Account", address.accountId.toEncodeObject()],
+        blockNumber,
+    );
+    if (data == null) {
```
Comment (sgkim126) [inline fee-monitor/src/Stake.ts]: Because we request the CCS balance of stakeholders, it's an error if it's null.

Comment (foriequal0) [reply fee-monitor/src/Stake.ts]: It can be a null according to the spec.

**Pull request merged**